ion_hash_tests.ion

```diff
@@ -0,0 +1,1631 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
remove 2017

Reply:
No need anymore as I am now using this file from the submodule.